### PR TITLE
[api] fix release behaviour when using token

### PR DESCRIPTION
**Pull request by adrianschroeter**
The token should also support the automatic time extension when
releasing into maintenance_release projects

### CODE REVIEW

**Review by krauselukas — APPROVED** [review]

**Comment by eduardoj** [comment]
I added a couple of commits to solve the RuboCop offenses, and another one to add the basis of the documentation of the newly introduced parameters.

Feel free to complete the documentation, modifying each TODO line and merging the commits. After that the PR should be ready to be reviewed.

File: src/api/app/models/token/release.rb [hunk 1]
```diff
@@ -5,27 +5,50 @@ def call(options)
...
   end
+
+  private
+
+  def release(package_to_release, source_repository, target_repository, time_now, options)
+    opts = { filter_source_repository: source_repository,
+             manual: true,
+             comment: 'Releasing via trigger event' }
+    opts[:multibuild_container] = options[:multibuild_flavor] if options[:multibuild_flavor].present?
+    opts[:filter_architecture] = options[:arch] if options[:arch].present?
```
Comment (krauselukas) [inline src/api/app/models/token/release.rb:36]: `filter_architecture` seems not to be used anywhere

Comment (krauselukas) [reply src/api/app/models/token/release.rb]: I removed it

**Comment by krauselukas** [comment]
@adrianschroeter since this is replaced by https://github.com/openSUSE/open-build-service/pull/13701 I would close this PR if that alright for you?

**Comment by adrianschroeter** [comment]
On Donnerstag, 19. Januar 2023, 17:21:52 CET Lukas Krause wrote:
> @adrianschroeter since this is replaced by https://github.com/openSUSE/open-build-service/pull/13701 I would close this PR if that alright for you?

sure

-- 

Adrian Schroeter ***@***.***>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH, Frankenstraße 146, 90461 Nürnberg, Germany
(HRB 36809, AG Nürnberg) Geschäftsführer: Ivo Totev

**Comment by danidoni** [comment]
Closing since this is replaced by https://github.com/openSUSE/open-build-service/pull/13701

**Pull request closed without merging**